scripts/jenkins/tags.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+case  $1  in
+    bare)
+        git tag | sort -V | tail -1 | tr -d '\n'
```

Comment:
This is up to you but I personally prefer using long options in scripts as they are more descriptive
```suggestion
        git tag | sort --version-sort | tail -1 | tr -d '\n'
```